internal/awsdata/dynamodb.go

```diff
@@ -0,0 +1,75 @@
+package awsdata
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/manywho/awsinventory/internal/inventory"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// AssetTypeDynamoDBTable is the value used in the AssetType field when fetching DynamoDB tables
+	AssetTypeDynamoDBTable string = "DynamoDB Table"
+
+	// ServiceDynamoDB is the key for the DynamoDB service
+	ServiceDynamoDB string = "dynamodb"
+)
+
+func (d *AWSData) loadDynamoDBTables(region string) {
+	defer d.wg.Done()
+
+	dynamodbSvc := d.clients.GetDynamoDBClient(region)
+
+	log := d.log.WithFields(logrus.Fields{
+		"region":  region,
+		"service": ServiceDynamoDB,
+	})
+
+	log.Info("loading data")
+
+	var tables []*string
+	done := false
+	params := &dynamodb.ListTablesInput{}
+	for !done {
+		out, err := dynamodbSvc.ListTables(params)
+
+		if err != nil {
+			d.results <- result{Err: err}
+			return
+		}
+
+		tables = append(tables, out.TableNames...)
+
+		if out.LastEvaluatedTableName == nil {
+			done = true
+		} else {
+			params.ExclusiveStartTableName = out.LastEvaluatedTableName
+		}
+	}
+
+	log.Info("processing data")
+
+	for _, t := range tables {
+		out, err := dynamodbSvc.DescribeTable(&dynamodb.DescribeTableInput{
+			TableName: t,
+		})
+		if err != nil {
+			d.results <- result{Err: err}
+			return
+		}
+
+		d.results <- result{
+			Row: inventory.Row{
+				UniqueAssetIdentifier:  aws.StringValue(out.Table.TableName),
+				Virtual:                true,
+				Location:               region,
+				AssetType:              AssetTypeDynamoDBTable,
+				HardwareMakeModel:      HumanReadableBytes(aws.Int64Value(out.Table.TableSizeBytes)),
```

Comment:
Would this be better as `Row.Comments`?

Reply:
EBS rows include the size as part of `HardwareMakeModel`, which is why I included it here, but I have no issue with putting it under `Comments`. No preference either way.

Reply:
Hmm thinking about it, I guess it's hardware information and we don't really have any better data to put here so should be fine to leave it. We can always change it in the future if it gets flagged in any FedRAMP audits

Reply:
Oh you already changed it=p Nevermind, it can stay in Comments